### PR TITLE
[master] updated in AUTOACCEPT MINIONS FROM GRAINS ,fixed #63708 and #61436

### LINKED ISSUE #63708: [DOCS] Need clarification in autoaccept_grains documentation about only working with strings
**Description**
When using autoaccept_grains the documentation does not state that the match will only for for grains that are strings.
It will not work for integers, lists of strings or Booleans as well. 

**Suggested Fix**
Note in documentation that the grain must be a string and give examples.

Here are some grains that wont work that are not custom grains.
efi-secure-boot - it's a Boolean
ipv4 - it's a list
server_id - It's an integer

**Type of documentation**
Current docs are here (https://docs.saltproject.io/en/latest/topics/tutorials/autoaccept_grains.html)

**Location or format of documentation**
https://docs.saltproject.io/en/latest/topics/tutorials/autoaccept_grains.html

**Additional context**

### PR DESCRIPTION
**Pull request by Tanmoy037**
### What does this PR do?
Update the AUTOACCEPT MINIONS FROM GRAINS docs . 
### What issues does this PR fix or reference?
Fixes: fixed #63708 and #61436

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [*] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by barbaricyawps — DISMISSED** [review]
@Tanmoy037 , this looks good to me from a communication standpoint. I'll make sure someone reviews it for technical accuracy.

File: changelog/63601.added.md [hunk 1]
```diff
@@ -0,0 +1 @@
+systemd daemon is reloaded, added in service state and module docs.
```
Comment (twangboy) [inline changelog/63601.added.md:1]: This seems different from the changes to the docs

**Review by twangboy — DISMISSED** [review]

File: changelog/63601.added.md (hunk 1 above)
Comment (Tanmoy037) [reply changelog/63601.added.md:1]: Yes, sorry I add this commit into this PR by mistake, It is from a different PR.

Comment (Tanmoy037) [reply changelog/63601.added.md:1]: Yes, sorry I added this commit to this PR by mistake, it was from a different PR.

File: doc/topics/tutorials/autoaccept_grains.rst [hunk 2]
```diff
@@ -42,5 +42,14 @@ Lines starting with a ``#`` are ignored.
     autosign_grains:
       - uuid
 
+Note that if you want to use custom grains or system grains that have a data type other than string, you need to encapsulate them in string format in the grains file on the minion:
```
Comment (Ch3LL) [inline doc/topics/tutorials/autoaccept_grains.rst:45]: This looks to be a bug that we should fix in the code, instead of document.

**Pull request closed without merging**